service/eventer/error.go

```diff
@@ -13,7 +13,9 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+var NotFoundError = microerror.New("not found")
+
 // IsNotFound asserts not found errors of eventer implementations.
 func IsNotFound(err error) bool {
-	return github.IsNotFound(err)
+	return microerror.Cause(err) == NotFoundError || github.IsNotFound(err)
```

Comment:
In the tests we need to be able to return some error the eventer can reason about. Exposing a not found error for the eventer interface is the only option I see here. So this is a necessary evil I can kind of live with.